### DISCUSSION TITLE
"Problem with the gridsquare set in your profile"

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#2954] [General]
"Distance worked" statistics page show the error message: "There is a problem with the gridsquare set in your profile"

![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/cad5b97d-bdc9-4099-9ca1-47f46e7419d0)

But the gridsquare is set correctly!
![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/9a9e2d93-2185-450e-bc1e-2244b11d4628)

![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/9964b4c4-4e1f-41a5-8049-36f10eb2c48c)

How to find reason of the error message?

**Comment by magicbug** [comment]
Error will be due to the giidsquare being greater than 6

**Reply by R1BLH** [reply]
Thank you!
This was the reason of the error message.
